tests/qweb/__snapshots__/qweb.test.ts.snap

```diff
@@ -1778,12 +1825,13 @@ exports[`t-on t-on with self and prevent modifiers (order matters) 1`] = `
     if (!context['onClick']) {
         throw new Error('Missing handler \\\\'' + 'onClick' + \`\\\\' when evaluating template 'test'\`)
     }
-    extra.handlers['click' + 2] = extra.handlers['click' + 2] || function (e) {if (e.target !== this.elm) {return}e.preventDefault();context['onClick'].call(owner, e);};
+    const handler3 = context['onClick'].bind(owner);
```

Comment:
hmm, it's annoying now that I see this: we create the binding every render, even though we only need it when it is not already present in the handlers object.

Would it not be better if the generated code would be something like this:
```js
  if (!extra.handlers['click' + 2]) {
   const handler3 = context['onClickPrevented'].bind(owner);
    extra.handlers['click' + 2] = function (e) {e.preventDefault();handler3(e);};
}
```